qcodes/instrument_drivers/Keysight/N9030B.py

```diff
@@ -0,0 +1,601 @@
+import numpy as np
+from typing import Any, Tuple, Dict, Union
+
+from qcodes import (
+    VisaInstrument, InstrumentChannel, Parameter, ParameterWithSetpoints
+)
+from qcodes.instrument.parameter import ParamRawDataType
+from qcodes.utils.validators import Enum, Numbers, Arrays, Ints
+from qcodes.utils.helpers import create_on_off_val_mapping
+
+
+class FrequencyAxis(Parameter):
+
+    def __init__(self,
+                 start: Parameter,
+                 stop: Parameter,
+                 npts: Parameter,
+                 *args: Any,
+                 **kwargs: Any
+                 ) -> None:
+        super().__init__(*args, **kwargs)
+        self._strt: Parameter = start
```

Comment:
Is there any particular reason why you changed `self._start` to `self._strt`  and similar for the other parameters?

Reply:
No, this has no particular reason here... I was just trying out and checking why the error was coming for freq_axis. I'll change it back.